abi/aapcs64/aapcs64.rst

```diff
@@ -466,11 +466,15 @@ Table 1, Byte size and byte alignment of fundamental data types shows the fundam
 Half-precision Floating Point
 -----------------------------
 
-The architecture provides hardware support for half-precision values. Two formats are currently supported: the format specified in IEEE 754-2008 and an Alternative format that provides additional range but has no NaNs or Infinities. This base standard of the AAPCS64 specifies two variants:
+The architecture provides hardware support for half-precision values. Three formats are currently supported:
 
-- The SysV-like variants use the IEEE 754-2008 defined format.
+1 - half-precision format specified in IEEE754-2008
```

Comment:
This numbering here, shouldn't it be done with some markup thing?

Reply:
Ah thanks, used the wrong syntax. Changed.